utils/vpr_pbtype_to_eblif.py

```diff
@@ -85,15 +123,26 @@ def main(args):
         args.pb_type
     )
 
-    pbtype_name, clocks, inputs, outputs = ports(pbtype_leaf)
+    blif_model, clocks, inputs, outputs, carry = ports(pbtype_leaf)
+    blif_model = pbtype_leaf.attrib['blif_model']
+
+    if 'subckt' in blif_model:
+        name = blif_model[len('.subckt '):]
```

Comment:
I'd rather use a regexp match. This will crash eg. if there is no explicit ".subkt " in the string.

Reply:
Fixed this and added some tests for this type of thing.